server/main.go

```diff
@@ -50,6 +50,15 @@ func Client(server *remotedialer.Server, rw http.ResponseWriter, req *http.Reque
 	defer resp.Body.Close()
 
 	logrus.Infof("[%03d] REQ OK t=%s %s", id, timeout, url)
+	for k, v := range resp.Header {
+		for _, h := range v {
+			if rw.Header().Get(k) == "" {
+				rw.Header().Set(k, h)
+			} else {
+				rw.Header().Add(k, h)
+			}
+		}
+	}
```

Comment:
I see your commit comment here, but what do you mean by "test server" ? Is server/main.go purely for testing purposes ? Checked in Rancher and looks like it's not referenced there. Client method being exported threw me off.

Reply:
Yes, both server/main.go and client/main.go are just examples or for testing, they aren't used in rancher. Updated the docs to mention that.